code/game/gamemodes/shadowling/special_shadowling_abilities.dm

```diff
@@ -33,6 +33,8 @@ var/list/possibleShadowlingNames = list("U'ruan", "Y`shej", "Nex", "Hel-uae", "N
 			for(var/obj/structure/alien/resin/wall/shadowling/R in shadowturf) //extremely hacky
 				qdel(R)
 				new /obj/structure/alien/weeds/node(shadowturf) //Dim lighting in the chrysalis -- removes itself afterwards
+			var/temp_flags = H.status_flags
+			H.status_flags |= GODMODE //Can't die while hatching
```

Comment:
@phil235 They already are not supposed to be able to die, that's sorta what the unbreakable walls are for, and its for any amount of other silly things that could cause death while hatching. Also going Plasmaman->Human->Shadowling is sorta silly, and doing the shadowling transform at the start makes no sense for the hatching process.